src/frontend/components/TeacherLiveLifecycleControls/index.spec.tsx

```diff
@@ -0,0 +1,265 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { videoMockFactory } from 'utils/tests/factories';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { liveState } from 'types/tracks';
+
+import { TeacherLiveLifecycleControls } from '.';
+import { StopLiveConfirmationProvider } from 'data/stores/useStopLiveConfirmation';
+
+jest.mock('jwt-decode', () => jest.fn());
+jest.mock('data/appData', () => ({
+  appData: { jwt: 'cool_token_m8' },
+}));
+
+describe('<TeacherLiveLifecycleControls />', () => {
+  it('renders info when you are not an administrator', () => {
```

Comment:
can you change `administrator` by `jitsi moderator` ? Because it is link to jitsi moderaot and not the role you have in the JWT token.